include/ocpp/v201/database_handler.hpp

```diff
@@ -65,13 +71,25 @@ class DatabaseHandler : public ocpp::common::DatabaseHandlerBase {
     /// \param id_token_hash
     void authorization_cache_delete_entry(const std::string& id_token_hash);
 
+    /// \brief Removes up to \p nr_to_remove items from the cache starting from the least recently used
+    ///
+    /// \param nr_to_remove Number of items to remove from the database
+    /// \retval True if succeeded
+    bool authorization_cache_delete_nr_of_oldest_entries(size_t nr_to_remove);
+
+    /// \brief Removes all entries from the cache that have an expiry date before \p now
+    ///
+    /// \param now The date to check the expiry for
+    /// \retval True if succeeded
+    bool authorization_cache_delete_entries_with_expiry_date_before(DateTime now);
```

Comment:
nit: I would either remove the argument and name the function `authorization_cache_delete_entries_with_expiry_date_before_now` or rename the argument, because theoretically this could be called with some other DateTime than now